spring:
  application:
    name: fzf-activity
  profiles:
    active: dev
  datasource:
      driver-class-name: com.mysql.jdbc.Driver
      type: com.zaxxer.hikari.HikariDataSource
      #将jdk1.8_0_333改为241版本的才不报连不上MySQL的错误，感觉333这个版本有问题
      jdbc-url: jdbc:mysql://localhost:3306/cbc_marketing?characterEncoding=UTF8&socketTimeout=60000&autoReconnect=true&allowMultiQueries=true&useSSL=true
      username: root
      password: fzf917
      minimum-idle: 5
      maximum-pool-size: 15
      auto-commit: true
      idle-timeout: 28800
      pool-name: DatebookHikariCP
      max-lifetime: 1800000
      connection-timeout: 28800
      connection-test-query: SELECT 1
#  task:
#    scheduling:
#      pool:
#        size: 5
  rabbitmq:
    addresses: 13.215.194.200:5672
    username: "admin"
    password: "admin"

server:
  port: 8819
mybatis:
  mapper-locations: classpath*:mapper/mysql/*.xml
  config-location: classpath:mybatis-config.xml

socket:
  port: 10005
  idleTime: 30000
  lvMessageLengthSize: 6

file:
  filePath: /Users/fanzhenfeng/tmp
  fileName: test.xlsx

schedule:
  task1: "-"
  task2: "-"






